libs/matdbg/src/TextWriter.cpp

```diff
@@ -402,13 +402,23 @@ static void printShaderInfo(ostream& text, const vector<ShaderInfo>& info,
     text << endl;
 }
 
-static bool printGlslInfo(ostream& text, const ChunkContainer& container) {
+static bool printGlslInfo(ostream& text, const ChunkContainer& container, ChunkType chunkType) {
     vector<ShaderInfo> info;
     info.resize(getShaderCount(container, ChunkType::MaterialGlsl));
-    if (!getGlShaderInfo(container, info.data())) {
+    if (!getGlShaderInfo(container, info.data(), ChunkType::MaterialGlsl)) {
```

Comment:
So it's not `chunkType` here and the line above?  

Also, maybe now that we're passing chunkType to `get*ShaderInfo`, maybe we can further consolidate these methods?  (If the `get*ShaderInfo` can be consolidated into one).

Reply:
Oops, missed this.